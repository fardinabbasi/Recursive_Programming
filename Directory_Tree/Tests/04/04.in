12 12
folder1 0
folder2 1
folder3 2
folder4 3
folder5 4
folder6 5
file1 6
file2 5
file3 4
file4 3
file5 2
file6 0
