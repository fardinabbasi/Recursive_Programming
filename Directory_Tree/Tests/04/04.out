|__folder1
|  |__folder2
|     |__folder3
|     |  |__folder4
|     |  |  |__folder5
|     |  |  |  |__folder6
|     |  |  |  |  |__file1
|     |  |  |  |__file2
|     |  |  |__file3
|     |  |__file4
|     |__file5
|__file6